### DISCUSSION TITLE
Show site admins/owners

### DISCUSSION THREAD

**Discussion by ellanrk** [microsoft-search/pnp-modern-search#3498] [Q&A] [answered]
Is there any property in this solution (Search Results webpart) that lists SharePoint site admins/owners in a column?

![image](https://github.com/microsoft-search/pnp-modern-search/assets/159083804/1a14e24a-bc5c-4aed-a72d-29c378a9f8f8)

**Comment by wobba** [comment, marked as answer]
No. Admin information is not available in the search index.
